src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -5686,6 +5690,7 @@ GetLastMethodIlOffset(
             return E_UNEXPECTED;
         }
 
+        delete[] Map;
```

Comment:
Doesn't this "delete" `MapLocal`?

Reply:
@mikedn Thank you for your comment. Yes, it does. And seeing how it's not initialized with new this will lead to an exception.

Reply:
@mikedn Resolved by reverting.